src/features/cseMachine/CseMachineLayout.tsx

```diff
@@ -149,10 +130,6 @@ export class Layout {
       v.reset();
     });
     Layout.values.clear();
-    Layout.compactValues.forEach((v, d) => {
-      v.reset();
-    });
-    Layout.compactValues.clear();
```

Comment:
Do we not need the `.forEach` in the new `Layout.values` anymore?

Reply:
Yes, the loop seems to be unnecessary, I'll remove it.